src/app/store.ts

```diff
@@ -1,15 +1,38 @@
-import { configureStore, ThunkAction, Action } from '@reduxjs/toolkit';
+import {
+  configureStore,
+  ThunkAction,
+  Action,
+  Store,
+  AnyAction,
+  ThunkDispatch,
+} from '@reduxjs/toolkit';
 // eslint-disable-next-line import/no-cycle
-import counterReducer from '../features/counter/counterSlice';
+import usersReducer, { UsersState } from '../features/users/userSlice';
+import postsReducer, { PostsState } from '../features/posts/postsSlice';
+import commentsReducer,
+{ CommentsState } from '../features/comments/commentsSlice';
 
-export const store = configureStore({
+export type AppThunkDispatch = ThunkDispatch<RootState, any, AnyAction>;
```

Comment:
Don't use any for types. Add the correct type here
![image](https://github.com/mate-academy/react_redux-list-of-posts/assets/70009205/e820808e-72af-4752-b536-3b4199facec9)